### PR TITLE
Fix receipt block propagation

### LINKED ISSUE #654: Cargo test behaves weirdly with devnet
this works:
`cargo test --color=always --package alphanet --lib tests`

and this does not:
`cargo test --color=always --package devnet --lib tests`
Error message:
```
running 1 test
error: Found argument 'tests' which wasn't expected, or isn't valid in this context
```

This test is introduced in https://github.com/nearprotocol/nearcore/pull/629

### PR DESCRIPTION
**Pull request by nearmax**
Also fixed #654

### CODE REVIEW

File: node/coroutines/src/ns_producer.rs [hunk 1]
```diff
@@ -35,7 +35,7 @@ pub fn spawn_block_producer(
     client: Arc<Client>,
     consensus_rx: Receiver<ConsensusBlockProposal>,
     mempool_control_tx: Sender<MemPoolControl>,
-    receipts_tx: Sender<ReceiptBlock>,
+    _receipts_tx: Sender<ReceiptBlock>,
```
Comment (ilblackdragon) [inline node/coroutines/src/ns_producer.rs]: let's remove this channels from alphanet and devnet then.

Comment (nearmax) [reply node/coroutines/src/ns_producer.rs]: Done.

File: node/devnet/src/lib.rs [hunk 2]
```diff
@@ -96,8 +80,12 @@ mod tests {
             std::fs::remove_dir_all(base_path.clone()).unwrap();
         }
 
-        let (mut client_cfg, devnet_cfg, rpc_cfg) = get_devnet_configs();
+        let mut client_cfg = configs::ClientConfig::default();
         client_cfg.base_path = base_path;
+        //    client_cfg.log_level = log::LevelFilter::Off;
```
Comment (ilblackdragon) [inline node/devnet/src/lib.rs]: add or remove?

File: node/alphanet/src/lib.rs [hunk 3]
```diff
@@ -46,10 +46,6 @@ pub fn start_from_client(
     let node_task = futures::lazy(move || {
         spawn_rpc_server_task(client.clone(), &rpc_cfg);
 
-        // Receipts from previous blocks.
-        let (receipts_tx, receipts_rx) = channel(1024);
-        spawn_receipt_task(client.clone(), receipts_rx);
```
Comment (ilblackdragon) [inline node/alphanet/src/lib.rs]: you should remove this function as well below

File: node/devnet/src/lib.rs (hunk 2 above)
Comment (nearmax) [reply node/devnet/src/lib.rs]: Added

File: node/alphanet/src/lib.rs (hunk 3 above)
Comment (nearmax) [reply node/alphanet/src/lib.rs]: Done.

**Pull request merged**